debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -1223,11 +1224,23 @@ protected String resolveCatalogName(String catalogName) {
         return catalogName;
     }
 
+    protected static CharSequence getEscapeSequence() {
```

Comment:
@vjuranek Is not this a bit overkill? Cant' we just use `String` constant `DEFAULT_ESCAPE_CHAR`?

Reply:
It's here for the purpose we can override it in DB specific sub-classes if there is any other escape char. But given that `\` is SQL standard, it's questionable if we will ever needed to use it. So yes, let's use simple approach and add possibility to override it when we really need it.